stable/victoria-metrics/values.yaml

```diff
@@ -0,0 +1,171 @@
+# Default values for victoria-metrics.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+nameOverride: ""
+fullnameOverride: ""
+clusterDomainSuffix: "cluster.local"
+
+vminsert:
+  replicaCount: 1
+
+  terminationGracePeriodSeconds: 60
+
+  extraArgs: {}
+
+  image:
+    repository: victoriametrics/vminsert
+    tag: latest
```

Comment:
It's better to stick some explicit version, to produce repeatable deployment.

Reply:
Specific version is pinned in values file which overrides default `values.yaml`.